profiles/kentik_snmp/citrix/netscaler-sdx.yml

```diff
@@ -0,0 +1,381 @@
+# https://mibs.observium.org/mib/SDX-ROOT-MIB/
+---
+extends:
+  - system-mib.yml
+
+provider: kentik-load-balancer
+
+sysobjectid:
+  - 1.3.6.1.4.1.5951.6    # NetScaler SDX root
```

Comment:
Is that what we want to see in the tags.kentik.model?

Reply:
that's how it's listed by Citrix; we can drop the `root` part if you prefer